README.md

```diff
@@ -173,27 +136,18 @@ saved:
 
 ## Requirements
 
-* Python 3.6 or newer
-* [Manticore](https://github.com/trailofbits/manticore/) release 0.2.2 or newer
-* [Flask](http://flask.pocoo.org/), which is used to run the JSON RPC server
+* Python 3.6 or newer 
```

Comment:
it's probably a good time to bump etheno to Python 3.7+, 3.6 is now EOL

Reply:
agreed - will make the change in the `setup.py` file as well